### ISSUE TITLE
Carregamento dos pontos no mapa do projeto com falha

### ISSUE THREAD

**Issue by elaste** [it3s/mootiro-maps#148]
No mapa do projeto não são carregados todos os pontos no mapa. http://maps.mootiro.org/project/307

**Comment by LuizArmesto** [comment]
Na verdade os objetos são carregados no mapa, mas não estão sendo exibidos os icones no mapa para objetos representados por polígonos, sendo que vários objetos no projeto do link estão como quadrados bem pequenos.

**Comment by elaste** [comment]
Ok, más acho isso confuso. Como podemos resolver. Realmente parece que os pontos não estão lá.

**Comment by LuizArmesto** [comment]
Sim, precisamos resolver. Meu comentário foi só para deixar registrado no issue o que está acontecendo de verdade no mapa, para ficar mais claro tanto sobre qual é o problema quanto para não precisar investigar novamente o bug quando for trabalhar na correção.

**Comment by LuizArmesto** [comment]
As possíveis correções são voltar a exibir o ícone em todos os objetos sempre, independente do tipo de geometria ou do zoom, ou fazer uma relação entre a área do objeto e o nível de zoom e decidir quando exibir ou não o ícone.

**Comment by elaste** [comment]
Difícil isso - já tivemos esse problema em outras ocasiões - quando o zoom é muito próximo, às vezes o ícone atrapalha e confunde, as pessoas não sabem se são dois objetos ou um só.

Sugestão: até o zoom de 1km exibimos os ícones e a partir de 500 exibimos áreas no caso de áreas (sem ícones) e os ícones no caso de pontos. 

Acho que devemos pensar em alguma forma de agrupamento análogo como fazemos com as comunidades no mapa: veja esse mapa: http://maps.mootiro.org/project/64/map - seria legal ter uma dimensão de quantas informações estão em qual região do mapa.
